src/main/java/racingcar/controller/RacingGameController.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.model.Cars;
+import racingcar.model.Racing;
+import racingcar.util.NumberGenerator;
+import racingcar.view.RacingGameView;
+
+public class RacingGameController {
+    private final NumberGenerator numberGenerator;
+    private final RacingGameView racingGameView;
+
+    public RacingGameController(NumberGenerator numberGenerator, RacingGameView racingGameView) {
+        this.numberGenerator = numberGenerator;
+        this.racingGameView = racingGameView;
+    }
+
+    public void startGame() {
+        List<String> carNames = racingGameView.inputCarNames();
+        int attempt = racingGameView.inputAttempt();
+        Cars cars = Cars.of(numberGenerator, carNames);
+        Racing racing = Racing.of(cars, attempt);
+
+        race(racing);
+
+        List<String> winnersNames = racing.winnersNames();
+        racingGameView.printWinner(winnersNames);
+    }
+
+    private void race(Racing racing) {
+        racingGameView.printGameStartMessage();
+        while (!racing.isFinish()) {
```

Comment:
개인적으로 !는 반대로 해석해야하기 때문에 racing의 종료 여부보다는 지속 여부를 판별하는 편이 가독성이 더 좋다고 생각합니다!

Reply:
처음에 isNotFinish 로 했다가 바꿨었는데, 말씀해신 대로 게임이 아직 진행중인가를 검증하는 메소드라는걸 더 명확히 표현해도 좋을거같네요!
```suggestion
        while (racing.isInProgress()) {
```
이정도면 괜찮을거같네요! 더 좋은 네이밍 있으면 의견 나눠봐요 ㅎㅎ

Reply:
충분히 좋은 네이밍입니다!